packages/client/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import * as RadixDialog from '@radix-ui/react-dialog'
+
+interface DialogProps {
+  open: boolean
+  onClose: () => void
+  children: React.ReactNode
+}
+
+const Dialog = (props: DialogProps) => {
+  const {open, onClose, children} = props
+  return (
+    <RadixDialog.Root
+      open={open}
+      onOpenChange={(newOpen) => {
+        if (!newOpen) {
+          onClose()
+        }
+      }}
+    >
+      <RadixDialog.Portal>
+        <RadixDialog.Overlay className='fixed inset-0 bg-slate-700/[.3] data-[state=open]:animate-overlayShow' />
+        {children}
+      </RadixDialog.Portal>
+    </RadixDialog.Root>
+  )
+}
+
+export default Dialog
```

Comment:
+1 I'd love to stop using default exports, they're harder to figure out for IDE and have other cons - [Why we have banned default exports in Javascript and you should do the same | by Kris Kaczor | Neufund](https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad)